src/openrct2/util/Util.cpp

```diff
@@ -351,17 +351,14 @@ int32_t strlogicalcmp(const char* s1, const char* s2)
     {
         if (*s2 == '\0')
             return *s1 != '\0';
-        else if (*s1 == '\0')
+        if (*s1 == '\0')
             return -1;
-        else if (!(isdigit(*s1) && isdigit(*s2)))
+        if (!(isdigit(*s1) && isdigit(*s2)))
         {
             if (toupper(*s1) != toupper(*s2))
                 return toupper(*s1) - toupper(*s2);
-            else
-            {
-                ++s1;
-                ++s2;
-            }
+            ++s1;
```

Comment:
Can you put a newline above? Makes it easier to read, especially since the if does not have braces.

Reply:
Sure thing. Funnily I had added one here but removed it later for consistency. I'll also add them to line 372 then.